helper/geojsonify.js

```diff
@@ -62,7 +62,12 @@ function search( docs, params ){
   var geojson = GeoJSON.parse( geodata, { Point: ['lat', 'lng'] });
 
   // add bbox
-  geojson.bbox = extent( geojson ) || undefined;
+  try {
+    geojson.bbox = extent( geojson ) || undefined;
```

Comment:
it looks a bit odd to have the `||` conditional inside a try/catch and after the thing that might throw. would be cleaner as

```javascript
geojson.bbox = undefined;
try {
  geojson.bbox = extent( geojson )
}
catch (e) {
  // ...
}